templates/README.md

```diff
@@ -0,0 +1,28 @@
+# Using templates to implement Secretless Connector Plugins
+
+We created connector templates to ease the process of adding new connectors to secretless.
+Before using the templates to add new connector plugins, be sure to read the [Secretless Connector Plugins README](https://github.com/cyberark/secretless-broker/blob/master/pkg/secretless/plugin/connector/README.md)
+
+To add a new connector do the following:
+
+1. Copy the relevant template directory (HTTP/TCP) into `internal/plugin/connectors/<protocol>`. 
+    1. Inside each template directory you will find the required files & structs implemented, with
+instructions to fill them with the content of the new connector
+1.  Add an entry to the `Plugins` map defined in GetInternalPluginsFunc() of `internal_plugins.go`, 
```

Comment:
this should be a link to a file, so the user doesn't have to search the code for this